docs/reference/ilm/ilm-and-snapshots.asciidoc

```diff
@@ -0,0 +1,26 @@
+[role="xpack"]
+[testenv="basic"]
+[[index-lifecycle-and-snapshots]]
+== Restoring Snapshots of Managed Indices
+
+beta[]
+
+When restoring a snapshot that contains indices managed by Index Lifecycle
+Management, the lifecycle will automatically continue to execute after the
+snapshot is restored. Notably, the `min_age` is relative to the original
+creation or rollover of the index, rather than when the index was restored.
+
+In cases where the index uses a policy with a Delete phase, this may cause the
+restored index to be deleted by Index Lifecycle Management immediately after it
+is restored. To prevent this, before restoring the snapshot,
+<<start-stop-ilm,lifecycle policy execution can be paused>> to allow the policy
+to be removed.
```

Comment:
I think we should re-word this to describe that in some case you may not want the lifecycle to continue to execute after the restore and in those case you should use the process described. This is for two reasons: 1) This is not a bug but the above wording may come across sounding like a bug to some users and 2) there are other reasons that having a delete action that you might not want the index to continue on its lifecycle, such as if it would allocate to a node attribute that no longer exists or if the policy no longer exists.

Reply:
Thanks - I've rewritten it to be clear that this is the intended behavior and extended it to other examples besides index deletion.